policy/policy.sample.json

```diff
@@ -0,0 +1,70 @@
+{
+    "SecurityPolicies": [
+        {
+            "Name": "policy1", # names can be same for multiple tenants thus use uuid.
```

Comment:
yes, but then we can't use database search feature like this https://github.com/romana/core/blob/master/tenant/store.go#L69 thus by resorting to complex find routines combining columns. this just keeps it simple.